config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml

```diff
@@ -603,6 +603,42 @@ presubmits:
       testgrid-tab-name: pr-secrets-store-csi-driver-e2e-akeyless
       description: "Run e2e test with akeyless provider for Secrets Store CSI driver."
       testgrid-num-columns-recent: '30'
+  - name: pull-secrets-store-csi-driver-e2e-windows-experimental
+    decorate: true
+    decoration_config:
+      timeout: 90m
+    always_run: false
+    path_alias: sigs.k8s.io/secrets-store-csi-driver
+    optional: true
+    branches:
+    - ^main$
+    labels:
+      preset-azure-cred: "true"
+      preset-azure-windows: "true"
+      preset-k8s-ssh: "true"
+      preset-dind-enabled: "true"
+      preset-azure-secrets-store-creds: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20231208-8b9fd88e88-master
+        command:
+          - runner.sh
+        args:
+          - bash
+          - -c
+          - >-
+            ./test/scripts/run-e2e-windows.sh
```

Comment:
I'll add the script in the driver repo and test with the `/test pull-secrets-store-csi-driver-e2e-windows-experimental` after this PR is merged.